src/Menu/Menu.scss

```diff
@@ -0,0 +1,51 @@
+$calorieTagColor: #49a0b5;
+$typeTagColor: #2e7cf6;
+$sideTagColor: #f7c244;
+
+.menu {
+    &.show-restaurants {
+        display: none;
+    }
+    &.show-menu {
+        display: block;
+    }   
+}
+
+.food-item-card {
+    border: solid 2px #898c8e;
```

Comment:
the colour here should probably be stored as a variable as well for consistency

Reply:
done!